src/components/NewPlantForm.js

```diff
@@ -1,17 +1,57 @@
-import React from "react";
+import React, { useState } from "react"
+import { v4 as uuid } from "uuid"
```

Comment:
Did the lab ask to use `uuid`?

Reply:
Nope! We used it in another lab to get unique id's, I've just started using it out of habit.

Reply:
Got it! If the lab doesn’t specifically ask for you to do that, please refrain. Part of the assessment grading is based on how well we follow directions, which includes sticking to what was asked. This is similar to avoiding scope creep as a developer. I love that you remember that and thought to add it though!